### PR TITLE
UI: Auto-focus input, form

### PR DESCRIPTION
**Pull request by utchoang**
### Description

This PR for fix bug "when opening a dialog, the first element should be active (with the cursor ready for typing). This avoids extra clicks."

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [x] Minor

#### Bug Severity

- [ ] BLOCKER
- [ ] Critical
- [ ] Major
- [x] Minor
- [ ] Trivial

### CODE REVIEW

**Comment by utchoang** [comment]
https://github.com/utchoang/cloudstack/pull/1 - Some dialogs do not have 'Cancel' buttons - for example 'Assign Instance to Another Account'

**Comment by utchoang** [comment]
https://github.com/utchoang/cloudstack/pull/2 - Create zone form is not aligned in mobile view

**Comment by utchoang** [comment]
@blueorangutan ui

**Comment by rhtyd** [comment]
@utchoang do you want this PR merged, as I see other PRs related to UI (asking if this one has additional changes or the other UI related PRs already address issues)?

**Comment by utchoang** [comment]
@rhtyd It can be merged if approved. Currently, I have separated PR so it is not relevant anymore.

**Comment by rhtyd** [comment]
@blueorangutan ui

File: ui/src/views/AutogenView.vue [hunk 1]
```diff
@@ -770,6 +778,16 @@ export default {
         this.fillEditFormFieldValues()
       }
     },
+    getFirstIndexFocus () {
```
Comment (davidjumani) [inline ui/src/views/AutogenView.vue:781]: Nice catch!

Comment (utchoang) [reply ui/src/views/AutogenView.vue:781]: @davidjumani What's wrong with this?

Comment (davidjumani) [reply ui/src/views/AutogenView.vue:781]: It's a good fix!

File: ui/src/views/offering/UpdateOfferingAccess.vue [hunk 2]
```diff
@@ -29,6 +29,7 @@
 
         <a-form-item :label="$t('label.domainid')" v-if="!this.offeringIsPublic">
           <a-select
+            :autoFocus="!this.isAdmin() && !this.offeringIsPublic"
```
Comment (davidjumani) [inline ui/src/views/offering/UpdateOfferingAccess.vue:32]:
```suggestion
            :autoFocus="!this.offeringIsPublic"
```

File: ui/src/views/offering/UpdateOfferingAccess.vue [hunk 3]
```diff
@@ -54,6 +55,7 @@
 
         <a-form-item :label="$t('label.zoneid')">
           <a-select
+            :autoFocus="!this.isAdmin() && this.offeringIsPublic"
```
Comment (davidjumani) [inline ui/src/views/offering/UpdateOfferingAccess.vue:58]:
```suggestion
            :autoFocus="this.offeringIsPublic"
```

File: ui/src/views/project/AddAccountOrUserToProject.vue [hunk 4]
```diff
@@ -31,7 +31,8 @@
             </span>
             <a-input
               v-decorator="['account']"
-              :placeholder="apiParams.addAccountToProject.account.description"/>
+              :placeholder="apiParams.addAccountToProject.account.description"
+              autoFocus />
```
Comment (davidjumani) [inline ui/src/views/project/AddAccountOrUserToProject.vue:35]: Another autofocus can be added at line 104

**Comment by davidjumani** [comment]
@utchoang Could you also add the autofocus to EditVM and ScaleVM

**Comment by davidjumani** [comment]
@blueorangutan ui

**Review by davidjumani — APPROVED** [review]
Tested LGTM!

**Review by rhtyd — APPROVED** [review]

**Pull request merged**